registry-index-service/index-manager.js

```diff
@@ -0,0 +1,374 @@
+const fs = require('fs');
+const process = require('process');
+const path = require('path');
+const codec = require('ripple-address-codec');
+const evernode = require('evernode-js-client');
+const { XrplApi, StateHelpers, RegistryClient, Defaults, RegistryEvents, } = require('evernode-js-client');
```

Comment:
Same package import two times. I feel we can avoid the 5th line and add those components to the 6th line.